### PR TITLE
`contrib`: `plotly` and `dash` subpackages

### PR DESCRIPTION
**Pull request by 0xMochan**
# `subgrounds.contrib`
A place to store extra addons of subgrounds, whether maintained or contributed.

## Changes
- `subgrounds.plotly_wrappers` deprecated in favor of `subgrounds.contrib.plotly`
- `subgrounds.dash_wrappers` deprecated in favor of `subgrounds.contrib.dash`

## Examples
Old examples set moved to `examples/dash_apps`
Stub examples (borrowed from docs) in root of `examples` (w/ link to docs for more examples).

In the future, we should throw some notebooks in here as well.

### CODE REVIEW

File: subgrounds/contrib/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Contrib is a niche concept in some libraries that represent extra / contributed content to a library that may not fit in the main package. This might be due to the maintainer not willing to maintain said content, the content being deemed too experimental, or perhaps it's unknown whether it's a "good idea" or not.
+
+> Relevant [Stackoverflow](https://softwareengineering.stackexchange.com/questions/252053/whats-in-the-contrib-folder) post
+
+For us, `subgrounds.contrib` will represent extra features and ideas with `subgrounds` that generally builds upon the core part of `subgrounds`. It allows us to add extensions or features to other libraries (such as `plotly`) without *relying* on the library as a dependency. We might add new concepts to this subpackage in the future, so look out!
+
+## What's currently here?
+
+### Plotly
+Originally located in `subgrounds.plotly_wrappers`, `subgrounds.contrib.plotly` contains helpful wrappers on `plotly` objects that allow you to use `FieldPaths` directly without creating a `panda`'s `DataFrame`.
```
Comment (cvauclair) [inline subgrounds/contrib/README.md:14]: Change "[...] creating a `panda`'s `DataFrame`" to "[...] creating a `pandas` `DataFrame`"

File: subgrounds/contrib/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+### Dash
+Originally located in `subgrounds.dash_wrappers`, `subgrounds.contrib.dash` contains helpful wrappers on `dash` objects that allow you to use other wrapped visualization objects (currently `subgrounds.contrib.plotly`) in `dash` apps without creating `panda`'s `DataFrame`s.
```
Comment (cvauclair) [inline subgrounds/contrib/README.md:17]: Same comment as above

File: examples/aave_v2.py
Comment (cvauclair) [inline examples/aave_v2.py:1]: Question which applies to most examples: It seems running most of the examples would not print anything, should we add a `print(df)` at the end? This way, users running e.g.: `python aave_v2.py` would at least see something

File: pyproject.toml [hunk 3]
```diff
@@ -50,6 +50,12 @@ version_toml = "pyproject.toml:tool.poetry.version"
 major_on_zero = false
 build_command = "poetry build"
 
+[tool.ruff]
```
Comment (cvauclair) [inline pyproject.toml:53]: `ruff` :muscle:

**Review by cvauclair — CHANGES_REQUESTED** [review]
Small typos I found in `subgrounds/contrib/README.md`, other comments are mostly just requests for clarification.

Nice work!

**Review by 0xMochan — COMMENTED** [review]
Resolved, thanks

**Pull request merged**